interop/relayer.go

```diff
@@ -119,16 +117,11 @@ func (r *L2ToL2MessageRelayer) Start(indexer *L2ToL2MessageIndexer, clients map[
 					}
 					// Pad gas by 33%.
 					paddedGas := (gasEstimate / 3) + gasEstimate
-					mu.Lock()
```

Comment:
this mutate does not work properly when multiple messages are sent in the same transaction. simplest way to fix this is to remove the lock and move the transactor within the case block